app/javascript/packages/verify-flow/steps/personal-key/personal-key-step.tsx

```diff
@@ -45,15 +46,27 @@ function PersonalKeyStep({ value }: PersonalKeyStepProps) {
       <DownloadButton
         content={personalKey}
         fileName="personal_key.txt"
+        onClick={() => {
+          trackEvent('IdV: download personal key');
+        }}
         isOutline
         className="margin-right-2 margin-bottom-2 tablet:margin-bottom-0"
       >
         {t('forms.backup_code.download')}
       </DownloadButton>
-      <PrintButton isOutline className="margin-right-2 margin-bottom-2 tablet:margin-bottom-0" />
+      <PrintButton
+        isOutline
+        onClick={() => {
+          trackEvent('IdV: print personal key');
+        }}
+        className="margin-right-2 margin-bottom-2 tablet:margin-bottom-0"
+      />
       <ClipboardButton
         clipboardText={personalKey}
         isOutline
+        onClick={() => {
+          trackEvent('IdV: copy personal key');
```

Comment:
It was easy enough to add this and the above (`print personal key`) events so I figured it wouldn't hurt to have some additional logging.